core/platforms/bsdi/bareos-dir.in

```diff
@@ -12,7 +12,7 @@ RETVAL=0
 case "$1" in
     start)
        echo "Starting the Bareos Director: "
-       @sbindir@/bareos-dir $2 -c @confdir@/bareos-dir.conf
+       @sbindir@/bareos-dir $2
```

Comment:
If bsdi means this: 
https://en.wikipedia.org/wiki/Berkeley_Software_Design
then we should remove the complete "bsdi" directory I think.

Reply:
Removed.